### ISSUE TITLE
[Bug]: Failed to create tray icon at startup warning

### ISSUE THREAD

**Issue by Sevastienn** [navossoc/MenuTools#5]
MenuTools produces this warning at the start of the system for no apparent reason. Despite this warning, MenuTools is running and working as intended.

![Screenshot 2020-12-03 161311](https://user-images.githubusercontent.com/44895281/101056035-7337f600-3582-11eb-8438-56abbfdc22c8.png)

**Comment by Sevastienn** [comment]
> MenuTools produces this warning at the start of the system for no apparent reason. Despite this warning, MenuTools is running and working as intended.
> 
> ![Screenshot 2020-12-03 161311](https://user-images.githubusercontent.com/44895281/101056035-7337f600-3582-11eb-8438-56abbfdc22c8.png)

It seems this warning can be avoided by setting a 30-second delay for the MenuTools task in task scheduler.

**Comment by navossoc** [comment]
What Windows are you using?

When you see the alert, did you see the icon on the tray icon?

TBH I need to do a review on this, but I've been very busy this past month.

**Comment by Sevastienn** [comment]
Thank you for the quick reply. This is definitely not something urgent.

As of the operating system, I run Windows 10 Build 20270.

As of the tray icon, it seems like an intermittent bug. Often the icon is created and you can see it in the tray despite the alert. Sometimes the icon is not created and MenuTools is indeed not running after the alert appears and sometimes there is no issue at all. That is what made me think there might be a conflict with some other program starting at startup of the computer and why I tried to delay the start of MenuTools by 30 seconds which seems to have completely resolved the issue.

**Comment by navossoc** [comment]
@Sevastienn yes, seems the issue is related on how Microsoft handles the "new initialization" of programs on the background.
Probably explorer is not ready yet, so that is the major issue.

**Comment by Sevastienn** [comment]
Yes, that seems likely. Anyway, the delay seems to be effective in avoiding the issue. By the way, thank you for an amazing piece of software!

**Comment by SlyAceZeta** [comment]
+1 I've been getting this issue too, but adding the 30-second delay in Task Scheduler did the trick. Thank you for mentioning that!

**Comment by fade2gray** [comment]
@Azekthi 
I always get that error when rebooting after a Win 10 update - another reboot and it's gone, but thanks for the heads-up 👍

**Comment by VitaliyAT** [comment]
visual cue
![image](https://github.com/navossoc/MenuTools/assets/1639608/38be2edd-f731-4589-a8e6-dfb73fb91e00)
